src/modules/ListItemMod.js

```diff
@@ -0,0 +1,127 @@
+};
+
+const createItem = (elem, classes, mealApi, xlink, likeApi,
+  callback0 = false, callback1 = false) => {
+  const docFrag = document.createDocumentFragment();
+  docFrag.appendChild(createImg(classes.image, mealApi.strMeal, mealApi.strMealThumb));
+  const elemHeader = createElementDefault('div', classes.titleHeader);
+  const titleImg = createElementDefault('p', classes.likesCtn, mealApi.strMeal);
+  elemHeader.appendChild(titleImg);
+  const elemLikes = createElementDefault('div', classes.likesIcon);
+  const elemLikesSVG = createNS(xlink.regular);
+  const elemLikesRedSVG = createNS(xlink.red);
+  elemLikesRedSVG.classList.add('hidden');
+  const elemLikesP = createElementDefault('p', classes.likesText, `${likeApi} like${likeApi > 1 ? 's' : ''}`);
+  elemLikes.append(elemLikesSVG, elemLikesRedSVG, elemLikesP);
+  elemHeader.appendChild(elemLikes);
+  docFrag.appendChild(elemHeader);
+  const btnDiv = createElementDefault('div', classes.btnCtn);
+  const commentBtn = createButton('button', classes.btnComment, 'Go to Comments', 'Comments');
+  if (callback1) commentBtn.addEventListener('click', callback1);
+  btnDiv.appendChild(commentBtn);
+  btnDiv.appendChild(createButton('button', classes.btnReserve, 'Go to Reservations', 'Reservations'));
+  docFrag.appendChild(btnDiv);
+  const itemElem = createElementDefault(elem, classes.container, false, docFrag);
+  itemElem.setAttribute('data-id', mealApi.idMeal);
+  if (callback0) itemElem.addEventListener('click', callback0);
+  return itemElem;
+};
+
+const createPopup = (elem, classes, mealApi, callback = false, callback2 = false) => {
+  const docFrag = document.createDocumentFragment();
+  const elemHeader = createElementDefault('div', classes.titleHeader);
+  const titlePop = createElementDefault('h2', classes.title, mealApi.strMeal);
+  const titleX = createElementDefault('span', classes.exit, 'X');
+  titleX.addEventListener('click', closePop);
+  elemHeader.append(titlePop, titleX);
+  docFrag.appendChild(elemHeader);
+  docFrag.appendChild(createImg(classes.image, mealApi.strMeal, mealApi.strMealThumb));
+  const elemProp = createElementDefault('div', classes.properties);
+  const mealName = createElementDefault('p', classes.property, `Name: ${mealApi.strMeal}`);
+  const mealCat = createElementDefault('p', classes.property, `Category: ${mealApi.strCategory}`);
+  const mealArea = createElementDefault('p', classes.property, `Area: ${mealApi.strArea}`);
+  const tag = mealApi.strTags ? mealApi.strTags.split(',').join(', ') : 'NONE';
+  const mealTags = createElementDefault('p', classes.property, `Area: ${tag}`);
+  elemProp.append(mealName, mealCat, mealArea, mealTags);
+  docFrag.appendChild(elemProp);
+  const secCtn = createElementDefault('div', classes.sectionCtn);
+  const formCtn = createElementDefault('div', classes.formContainer);
+  if (callback) secCtn.appendChild(callback);
+  if (callback2) formCtn.appendChild(callback2);
+  docFrag.append(secCtn, formCtn);
```

Comment:
- You have done a great job💪 making the pop-up for comments work. However,🙆‍♂️ the pop-up doesn't fit well on the screen🖥 and overflows to the extent that it's difficult😂 to find the submit button. 🙏Please adjust the styling to ensure it fits perfectly on the screen🖥. If it's not possible to fit it entirely, at least make it scrollable so that users👨‍👩‍👧‍👦 can easily find the submit button. This will greatly enhance the user👩‍🦰 experience when adding a comment to a meal.

Your pop-up:
![image](https://github.com/lordksix/Meals-API-template/assets/63445083/63fffd80-5448-4df5-8793-db6ec24cfbc4)

Reply:
Thank you for the fix back. It was fixed in commits f8856be2a5960e801cf9837982a5934b44a067b0 and f164e434fdbf4dd2b21685aa142ac9136aacff32